themes/finna2/templates/Helpers/R2RestrictedRecordPrompt.phtml

```diff
@@ -0,0 +1,19 @@
+<div class="r2-restricted-record-register alert alert-info">
+  <div class="r2-restricted-info">
+    <div class="icon"><i class="fa fa-research fa-2x"></i></div>
+  </div>
+  <div class="r2-restricted-content">
+    <?php if ($this->warning): ?>
+      <div><?= $this->translate($this->warning) ?></div>
```

Comment:
Onko nämä HTML:ää? Kommentti mukaan, jos jätetään tarkoituksella escapettaamatta. Tyyliin `// intentionally not escaped so that markup can be used `. Sama  juttu kaikkiin kohtiin joissa näin toimitaan.

Reply:
Käänös ei sisällä enää html:ää.